fixtures/assertion-removal/src/render.tsx

```diff
@@ -13,7 +13,7 @@ const skuRender: Render = {
       </SkuProvider>,
     ),
 
-  renderDocument: ({ app, headTags, bodyTags }) => dedent`
+  renderDocument: ({ app, headTags, bodyTags }) => dedent/* html */ `
```

Comment:
Pro tip: if you alias `dedent` to `html` Prettier will format the markup inside the string tag, [like this](https://prettier.io/playground/#N4Igxg9gdgLgprEAuc0DOMAEALGBbAG0wF5MATOC2AbgB0p7IoNMAjAgQ2xJ3wIHoAVL0KZB-AAb0APAEIAIgHkAwgBUAmgAUAoiIIA+epmOZpuQoagmTZuBzKXrmWjBc28cGB0xhsHAE5onsS0IACqqgBiALQAHKGOTsbSMACWMARw+gCyAJ6YAIIA7nBoEB6Ymv4QAFZwYDDS-GkZWUZJph5emFAcHiEgAG6pcEUADhD+riA+0PCwA0WpZDDYxBTDYHDRSyvYADSYqVDpqRwE0Whg53DEAIwJ7UkAJMDYdmSqHADmaAC+TXe9kS1mkrAgZFyIKc0jIqUGRzIAw4YzGCVeKLGAP4cMG0Osr3BkK+vz+Txs-CJUPJgL4lgk1BA+xAEDGaXQyFAAWqRU0AQQaGQIHORQ4uUFzNY-g4YAA1p4AMp9OAAGWOcGQADNzkFJdK5YqxjLjt9kDB-ABXODMuB4ViUChkFUcKDfC0-OCRSZ4DgwNKuoUcC0wCBMkDmAgAdWw6VKRq2CoFp2GMFyQrAaAlIGOQSmVR+Pq1OutIBqaAAHgqTZkAIoWiDwIsEXUgI2BOD+IWsDj2ghhsb+Y4wSPLVbIWIABmZA4gQUj0rGQoHpQ7gw1zIAjvX4FVWYKUBw0NEoHBKJQw-44FvUpf899C0htc2S0E8KkzZaX9W4HWGxrH8WzJeKwI57MgABMQHSqkBAmso5QPiApQAKxhhaQRfKw+5Pi2gxWgAklAVAwAqYCDmyBREQqqaZE2QR-H8QA).

Reply:
That's neat.